ffi/src/prove.rs

```diff
@@ -1,27 +1,44 @@
 use crate::utils::{c_char_to_str, c_char_to_vec, vec_to_c_char};
 use libc::c_char;
-use prover::{utils::init_env_and_log, zkevm};
+use prover::{aggregator, config::ALL_AGG_DEGREES, utils::init_env_and_log, zkevm};
 use std::cell::OnceCell;
 use types::eth::BlockTrace;
 
-static mut PROVER: OnceCell<zkevm::Prover> = OnceCell::new();
+static mut CHUNK_PROVER: OnceCell<zkevm::Prover> = OnceCell::new();
+static mut AGG_PROVER: OnceCell<aggregator::Prover> = OnceCell::new();
+static mut AGG_CHUNK_TRACES: OnceCell<Vec<Vec<BlockTrace>>> = OnceCell::new();
```

Comment:
remove this line

Reply:
Delete is in commit [fc94edf](https://github.com/scroll-tech/scroll-prover/pull/160/commits/fc94edf7f55a250b3a51eb298a21715857f8db6d).